frontend/src/services/settings.ts

```diff
@@ -115,34 +92,9 @@ export const maybeMigrateSettings = async (logout: () => void) => {
   if (currentVersion < 4) {
     logout();
   }
-
-  if (currentVersion < 5) {
-    const localSettings = getLocalStorageSettings();
-    await saveSettings(localSettings);
-  }
```

Comment:
why did we lose this migration?

Reply:
My mistake, I misread it as something else. I have since moved the settings migration logic to its own hook because it was utilizing some methods provided by other hooks.

I have also moved the logic away from being executed after saving settings to avoid complication (since the function needs to call save settings, the alternative solution was POST save settings -> migrate -> POST save settings again)